package.json

```diff
@@ -47,6 +45,5 @@
     "phantomjs-svg2png.js",
     "lru-cache.js",
     "logo"
-  ],
-  "engines": { "node": "0.10.x" }
```

Comment:
I think we could upgrade it, but why remove it? What is the first version of node which supports Promise?